base/src/main/java/io/spine/query/AbstractQueryBuilder.java

```diff
@@ -161,6 +161,19 @@ public final B withMask(Field...fields) {
         ImmutableList<String> paths = Arrays.stream(fields)
                                               .map(Field::toString)
```

Comment:
Please fix the alignment. Consider having `Arrays.stream()` coming on the new line after `=`.